deepchem/models/torch_models/layers.py

```diff
@@ -4792,3 +4792,86 @@ def forward(self, dist: torch.Tensor) -> torch.Tensor:
         dist = dist.unsqueeze(-1) / self.cutoff
         output: torch.Tensor = self.envelope(dist) * (self.freq * dist).sin()
         return output
+
+
+class EncoderRNN(nn.Module):
+    """Encoder Layer for SeqToSeq Model.
+
+    It takes input sequences and converts them into a fixed-size context vector
+    called the "embedding". This vector contains all relevant information from
+    the input sequence. This context vector is then used by the decoder to
+    generate the output sequence and can also be used as a representation of the
+    input sequence for other Models.
+
+    Examples
+    --------
+    >>> from deepchem.models.torch_models.layers import EncoderRNN
+    >>> import torch 
+    >>> embedding_dimensions = 7
+    >>> num_input_token = 4
+    >>> input = torch.tensor([[1, 0, 2, 3, 0], [0, 0, 0, 0, 0], [0, 0, 0, 0, 0]])
+    >>> layer = EncoderRNN(num_input_token, embedding_dimensions)
+    >>> emb, hidden = layer(input)
```

Comment:
may want to show the length of the embedding is 7

Reply:
Done